.github/workflows/build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node-version: [ 16.x, 18.x, 20.x ]
+        node-version: [ 18.x, 20.x ]
```

Comment:
Probably should be 20, 22, and 24. 18 is no longer supported by AWS and 24 will be coming into the rotation in November so we can get ahead of it.